suite.go

```diff
@@ -313,8 +308,20 @@ func (suite TestSuite) compareObjects(obj map[string]interface{}, expectedObj ma
 			}
 		}
 	}
-	// Deep compare the objects and return all errors
-	return deep.Equal(obj, expectedObj)
+
+	// Deep compare the objects and return any errors on non-ignored fields
+	diffs := make([]string, 0)
```

Comment:
Can you add a comment that states that this does a post-process removal of `ignoredFields` because we're using the deep lib.